thrust/device_vector.h

```diff
@@ -97,7 +97,7 @@ template<typename T, typename Alloc = thrust::device_malloc_allocator<T> >
      *  \param v The \p device_vector to copy.
      */
     template<typename OtherT, typename OtherAlloc>
-    __device__
+    __device__ explicit
```

Comment:
Why was explicit added here? It will likely break user code if device_vector cannot be assigned from other vectors.